mcux/middleware/usb/device/usb_device_ehci.c

```diff
@@ -6,7 +6,7 @@
  * SPDX-License-Identifier: BSD-3-Clause
  */
 
-#define DT_DRV_COMPAT nxp_kinetis_usbd
+#define DT_DRV_COMPAT nxp_mcux_usbd
 
```

Comment:
Was this driver initially used for Kinetis based USBD?

Reply:
No, it was used for MXRT devices only.